### ISSUE TITLE
npm install fails

### ISSUE THREAD

**Issue by snowPu** [stackgl/headless-gl#267]
Environment: Ubuntu, python 3.8.10, node 16.17.0

```
> gl@7.0.0 install
> prebuild-install || node-gyp rebuild

prebuild-install warn install No prebuilt binaries found (target=16.17.0 runtime=node arch=x64 libc= platform=linux)
gyp info it worked if it ends with ok
gyp info using node-gyp@9.4.0
gyp info using node@16.17.0 | linux | x64
gyp info find Python using Python version 3.8.10 found at "/usr/bin/python3.8"
gyp info spawn /usr/bin/python3.8
gyp info spawn args [
gyp info spawn args   '/home/sukanya/X/Navlite/headless-gl/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'make',
gyp info spawn args   '-I',
gyp info spawn args   '/home/sukanya/X/Navlite/headless-gl/build/config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/home/sukanya/X/Navlite/headless-gl/node_modules/node-gyp/addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/home/sukanya/.cache/node-gyp/16.17.0/include/node/common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=/home/sukanya/.cache/node-gyp/16.17.0',
gyp info spawn args   '-Dnode_gyp_dir=/home/sukanya/X/Navlite/headless-gl/node_modules/node-gyp',
gyp info spawn args   '-Dnode_lib_file=/home/sukanya/.cache/node-gyp/16.17.0/<(target_arch)/node.lib',
gyp info spawn args   '-Dmodule_root_dir=/home/sukanya/X/Navlite/headless-gl',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.'
gyp info spawn args ]
/bin/sh: 1: python: not found
gyp: Call to 'python commit_id.py check ..' returned exit status 127 while in angle/src/angle.gyp. while loading dependencies of binding.gyp while trying to load binding.gyp
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/home/sukanya/X/Navlite/headless-gl/node_modules/node-gyp/lib/configure.js:325:16)
gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:291:12)
gyp ERR! System Linux 5.15.0-84-generic
gyp ERR! command "/usr/local/bin/node" "/home/sukanya/X/Navlite/headless-gl/node_modules/.bin/node-gyp" "rebuild"
gyp ERR! cwd /home/sukanya/X/Navlite/headless-gl
gyp ERR! node -v v16.17.0
gyp ERR! node-gyp -v v9.4.0
gyp ERR! not ok 
npm ERR! code 1
npm ERR! path /home/sukanya/X/Navlite/headless-gl
npm ERR! command failed
npm ERR! command sh /tmp/install-c386ff5c.sh

```

**Comment by AEPKILL** [comment]
Maybe you need install python@2

```sh
/bin/sh: 1: python: not found
```

**Comment by NotReeceHarris** [comment]
```
sudo apt install python-is-python3
```

or put this in your bashrc

```
alias python=python3
```

**Comment by dhritzkiv** [comment]
The latest version is available as [v8.0.2](https://github.com/stackgl/headless-gl/releases/tag/v8.0.2) and is prebuilt for linux/macOS.
